src/main/features/response/routes/when-will-you-pay.ts

```diff
@@ -0,0 +1,53 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { ErrorHandling } from 'common/errorHandling'
+import { DefendantPaymentOption, DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { User } from 'idam/user'
+import { DraftService } from 'services/draftService'
+
+function renderView (form: Form<DefendantPaymentOption>, res: express.Response) {
+  const user: User = res.locals.user
+  res.render(Paths.defenceFullPartialPaymentOptionsPage.associatedView, {
+    form: form,
+    claim: user.claim
+  })
+}
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.defenceFullPartialPaymentOptionsPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+      const user: User = res.locals.user
+      const defendantPaymentOption: DefendantPaymentOption = user.responseDraft.document.defendantPaymentOption
```

Comment:
I don't think we need two variables, `user` would be enough to get type checking.

Reply:
Did try and use just one variable but it didn't like it, res.locals.user.responseDraft.document.defendantPaymentOption

Reply:
Had no problem with it.